### ISSUE TITLE
Cannot find .tar.gz package when submitting Python job

### ISSUE THREAD

**Issue by chelxom** [Microsoft/vs-tools-for-ai#44]
*Issue:*
When submitting a Python job, Tools for AI showed error message that some `{GUID}.tar.gz` package couldn't be found in `%TEMP%\8\ToolsForAITemp`. 
This folder path was also set in VS Options as temporary directory of AI Tools.

After resetting AI Tools option, the `8` in temporary directory above became `4`, and it was possible to submit Python job.

*Environment:*
OS: Windows 8
IDE: VS2017

**Comment by linmajia** [comment]
@chelxom , thank you very much for the feedback. Sorry for the late response.

By default, Tools for AI uses %Temp%\ToolsForAITemp to store temporary data. There won't be any intermediate path like "8" or "4". I guess that %TEMP%\8\ToolsForAITemp may not exist and so it causes the error.

**Comment by chelxom** [comment]
@linmajia , yes the folder whose path has "8" didn't physically exist when submit failed for the first time. And in AI Tools option page, the temporary directory was set to `%TEMP%\8\ToolsForAITemp`. 
It seemed the same like that a compressed package wasn't created (However it might be that folder hierarchy user typed wasn't created).

And to be clear, it was the path shown in option page became `%TEMP%\4\ToolsForAITemp` after resetting temporary directory in AI Tools option page. This might seem inconsistent while practically didn't cause error.
